<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CannyDialog</class>
 <widget class="QDialog" name="CannyDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>211</width>
     <height>158</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QCheckBox" name="CannyCheckBox">
      <property name="text">
       <string>Apply Blur</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="CannyLtwoCheckBox">
      <property name="text">
       <string>L2 Gradient</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="CannyKernelSpinBox">
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="CannyCancelButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Min Threshold</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Kernel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="CannyThreshSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Max Threshold</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="CannyMaxThreshSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
